pkg/reader/ccip.go

```diff
@@ -655,7 +658,7 @@ func (r *ccipChainReader) getOfframpStaticConfig(
 //nolint:lll // It's a URL.
 type offrampStaticChainConfig struct {
 	ChainSelector      cciptypes.ChainSelector `json:"chainSelector"`
-	RmnProxy           []byte                  `json:"rmnProxy"`
+	Rmn                []byte                  `json:"rmn"`
```

Comment:
Hmm tests seem to be failing after this change, maybe i was wrong?

Reply:
The RMNRemote contract is not defined in the dest chain reader defintion. This PR should fix it: https://github.com/smartcontractkit/chainlink/pull/14588